CONTRIBUTING.md

```diff
@@ -266,6 +266,44 @@ tokio-console http://127.0.0.1:5555
 
 </details>
 
+### Profiling
+
+<details> <summary> Expand to learn ho to profile iroha. </summary> </details>
+
+To optimize performance it's useful to profile iroha.
+
+To do that you should compile iroha with `profiling` profile and with `wasm_profiling` compilation flag: 
+
+```bash
+RUSTFLAGS="-C force-frame-pointers=on --cfg wasm_profiling"cargo build --profile profiling
+```
+
+Then start iroha and attach profiler of your choice to the iroha pid.
+
+Alternatively it's possible to build iroha inside docker with profiler support and profile iroha this way.
+
+```bash
+docker build -f Dockerfile.glibc --build-arg="PROFILE=profiling" --build-arg='RUSTFLAGS=-C force-frame-pointers=on --cfg wasm_profiling' -t iroha2:profiling .
+```
+
+E.g. using perf (available only on linux):
+
+```bash
+# to capture profile
+sudo perf record -g -p <PID>
+# to analyze profile
+sudo perf report
+```
+
+It's also possible to profile iroha's wasm executor to do then unstriped version should be provided.
```

Comment:
I think it should be mentioned that profiling the wasm executor can be done using the same tools as profiling the iroha itself (maybe even link to [wasmtime's docs on the subject](https://docs.wasmtime.dev/examples-profiling.html)?).

It's not obvious that this will work, as it actually requires some support from wasmtime and mentioning this will lead to less confusion IMO.

Reply:
I’ve ment that we could see what is going on inside executor while we run iroha. 
Not that we should invoke some external command.
I will try to make this part more clear.